osbs/api.py

```diff
@@ -615,7 +615,7 @@ def resume_builds(self, quota_name=None):
 
     # implements subset of OpenShift's export logic in pkg/cmd/cli/cmd/exporter.go
     @staticmethod
-    def _prepare_resource(resource_type, resource):
+    def _prepare_resource(resource):
```

Comment:
I wasn't sure if this was intended for future use, but I guess it's not an API so can be removed.